src/tests/tracing/eventpipe/common/BinaryWriterExtensions.cs

```diff
@@ -0,0 +1,25 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Tracing.Tests.Common
+{
+    internal static class BinaryWriterExtensions
+    {
+        public static void WriteString(this BinaryWriter @this, string value)
```

Comment:
Do we really need this? It's not clear to me why message contents matter here, as `ValidateResponseMessage` is only concerned with the responseMessage's header. And so if contents don't actually matter for these tests, we can remove this and reduce the code introduced even further.

It also feels like `@this` is wrong.